libs/sdk-py/langgraph_sdk/client.py

```diff
@@ -579,7 +1107,91 @@ async def create(
         webhook: Optional[str] = None,
         multitask_strategy: Optional[MultitaskStrategy] = None,
     ) -> Run:
-        """Create a background run."""
+        """Create a background run.
+
+        Args:
+            thread_id (optional,str): the thread ID to assign to the thread. Defaults to None.
+                If None a random UUID will be generated.
+            assistant_id (str): The assistant ID or graph name to stream from.
+                If using graph name, will default to first assistant created from that graph.
+            input (optional, dict): The input to the graph. Defaults to None.
+            metadata (optional, dict): Metadata to assign to the run. Defaults to None.
+            config (optional, Config): The configuration for the assistant. Defaults to None.
+            checkpoint_id (optional, str): The checkpoint to start streaming from. Defaults to None.
+            interrupt_before (optional, list[str]): Nodes to interrupt streaming on immediately before they get executed.
```

Comment:
`Nodes to interrupt streaming on immediately before they get executed.` can we change this to just `Nodes to interrupt immediately before they get executed.`

streaming is confusing in the context of a background run